### DISCUSSION TITLE
有没有简厨

### DISCUSSION THREAD

**Discussion by woshijiandegou** [share121/inter-knot#2213] [其他]
![1723696038745](https://github.com/user-attachments/assets/be4b961d-50c2-48d9-8043-ae0980179ad3)
我爱简！

**Comment by Ankoblue** [comment]
111

**Comment by fengqyb** [comment]
i了i了

**Comment by Hongxuli2006** [comment]
当然有

**Comment by Will-Dyt** [comment]
111

**Comment by biliqide** [comment]
😍

**Comment by wugan-bot156** [comment]
[看看简的这个视频](https://www.bilibili.com/video/BV1Nx4y147PA/?spm_id_from=333.337.search-card.all.click&vd_source=918cb1ee8485833a3f5916436c4caf8e)

**Comment by Phaethon-0019** [comment]
我好喜欢晚上她用她的尾巴把我捆起来

**Comment by daofeng9612** [comment]
我我我我我我我·

**Comment by duskbirds** [comment]
表情丰富传神，肢体动作独特且符合角色人设，再加上绝美的建模。<img src="https://bbs-static.miyoushe.com/static/2023/04/28/ad3e4d3d37302d240339f611b2458ef9_7921317972787298019.png" alt="米仔_比心" width="75" >
[务必看看这个](https://www.bilibili.com/video/BV15keteUEHA/?spm_id_from=333.1007.top_right_bar_window_history.content.click&vd_source=021a3e6832ecf139021f60200afb8860)

**Comment by BitterWind** [comment]
试试看表情包
<img src="https://bbs-static.miyoushe.com/static/2024/07/31/297dfcb691d834ce4960541412484f43_1488965655979817584.png" alt="椒丘 撒椒" width="75" >

**Comment by edge-stroke** [comment]
11111

**Comment by March-777th** [comment]
555恨不得把蛋塞进去<img src="https://bbs-static.miyoushe.com/static/2024/06/19/8079e2c46c0698178c735198f9340cff_3596969066154449614.png" alt="花火 礼物" width="75" >

**Comment by Bai6869** [comment]
111

**Comment by shangjin2001** [comment]
我！

**Comment by wwwsheepwww** [comment]
可惜给青衣榨干了

**Comment by LuAn-afl** [comment]
我不是简厨我只是馋她身子要是让我小保底抽到简我天天给她洗袜子都行

**Comment by ytny1** [comment]
我我我<img src="https://bbs-static.miyoushe.com/static/2024/07/31/297dfcb691d834ce4960541412484f43_1488965655979817584.png" alt="椒丘 撒椒" width="75" >

**Comment by chihung3927** [comment]
([看看这个](https://www.bilibili.com/video/BV1hW4219758/?spm_id_from=333.788&vd_source=5b684108f2eb1ba79dad0d258982bf15))

**Comment by Ruanachan** [comment]
我简爱

**Comment by AIRFLYSKY** [comment]
废话，怎么可能没有，我就是简厨<img src="https://bbs-static.miyoushe.com/static/2023/11/16/810dd11a4d16759f16ca078596aab6b1_422007669816866994.png" alt="哲-让我想想" width="75" >

**Comment by ALXY-12** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by LilangAlan** [comment]
<img src="https://upload-bbs.miyoushe.com/upload/2023/03/09/3173bc3e69fcfebecfda741340c9ad22_6841250310756275678.png" alt="开拓者 吃瓜ing" width="75" >

**Comment by 15666851105** [comment]
那必然是简！！！！！！（阴暗扭曲的疯狂爬行~~）

**Comment by waerte-yang** [comment]
简
![1723640484869](https://github.com/user-attachments/assets/56fca399-d792-4868-abf9-cb1325974c5e)

**Comment by alannnnnn22** [comment]
猫又简双厨报道！

**Comment by RaphaelHider** [comment]
看完前瞻毫无感觉，过完主线被彻底征服，这是我抽星见雅之前会抽的最后一个限五！

**Comment by OneMiaomiao** [comment]
![93d770a3a5c9175fc8d00e25492595d](https://github.com/user-attachments/assets/3c77fbd7-9405-4955-a273-75e9e4faf63b)
